templates/CRM/Activity/Form/Activity.tpl

```diff
@@ -179,7 +179,6 @@
       {if $action eq 4}
         {include file="CRM/Custom/Page/CustomDataView.tpl"}
       {else}
-        <div id="customData"></div>
```

Comment:
Why is this being removed?

Reply:
OK - I can see it duplicated in the include file. I'm not wild about it being included with an unrelated change without notes but I think it makes sense.